itowns/index.js

```diff
@@ -72,6 +72,13 @@ async function main() {
 
     const overviews = await getOverviews;
 
+    // on ajoute les dataset.limits pour les layers graph/contour
+    // avec uniquement les niveaux correspondants au COG mis à jour par les patchs
+    overviews.dataSet.limitsForGraph = {};
+    for (let l = overviews.dataSet.level.max - 4; l <= overviews.dataSet.level.max; l += 1) {
```

Comment:
Il faudrait sortir le 4 comme fait dans le calcul du cache :
`def get_slabdeepth(slabSize):`
`    """Get the number of levels per COG"""`
`    slabsize = min(slabSize['width'], slabSize['height'])`
`    return math.floor(math.log(slabsize, 2)) + 1`

Reply:
en effet, je vais faire la correction